src/services/ui/src/pages/dashboard/index.tsx

```diff
@@ -33,11 +35,20 @@ const loader = (queryClient: QueryClient) => {
     return isUser;
   };
 };
+
 export const dashboardLoader = loader;
 
 export const Dashboard = () => {
-  const { data: user } = useGetUser();
+  const navigate = useNavigate();
+  const userContext = useUserContext();
   const query = useOsQuery();
+  const { data } = useGetUser();
+
+  useEffect(() => {
+    if (data?.user?.["custom:cms-roles"] === "onemac-micro-statesubmitter") {
+      return navigate("/");
```

Comment:
Within the `navigate()` arguments, we should be using the `ROUTES` constant instead of string paths

Reply:
Navigate is still using the hard string instead of `ROUTES.HOME`